code/datums/world_topic.dm

```diff
@@ -141,7 +141,6 @@
 /datum/world_topic/status/Run(list/input)
 	. = list()
 	.["version"] = GLOB.game_version
-	.["mode"] = GLOB.master_mode
```

Comment:
I assume banner code and such were changed ?

Reply:
After I complained about it for 2 weeks, yes 😄

Reply:
This was TM'd and Sybil reported as down, but then it got fixed.